scripts/run-e2e-job.sh

```diff
@@ -0,0 +1,11 @@
+set -o errexit
+set -o nounset
+set -o pipefail
+
+MARKETPLACE_OPERATOR_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+# Get operator-sdk binary.
+wget -O /tmp/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.2.0/operator-sdk-v0.2.0-x86_64-linux-gnu && chmod +x /tmp/operator-sdk
+wget -O /tmp/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.11.3/bin/linux/amd64/kubectl && chmod +x /tmp/kubectl 
```

Comment:
Suggestion: Version number of k8s and sdk can be fed into the URLs from variables defined in the script.